dv/uvm/icache/dv/Makefile

```diff
@@ -0,0 +1,40 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Simple wrappers around dvsim.py to avoid needing long command lines. This
+# doesn't do any dependency tracking (as you might expect in a Makefile) because
+# the dvsim infrastructure doesn't support it.
+
+# Set WAVES to 1 to get VPD/FSDB dumps
+WAVES=0
+
+# Set VERBOSITY to a nonempty string and it will be passed to dvsim with the
+# --verbosity parameter, overriding whatever was set in the hjson file.
+VERBOSITY=
+
+# Specify the seed for the test to run. If this is empty, dvsim.py will pick
+# random seeds. By default, we make runs reproducible, so force the seed.
+SEED=1
+
+ibex-top       := ../../../..
+dvsim-py       := $(ibex-top)/vendor/lowrisc_ip/dvsim/dvsim.py
+dvsim-std-args := --skip-ral --scratch-root $(ibex-top)/build -v h
+
+waves-arg      := $(if $(filter-out 0,$(WAVES)),--waves,)
+verbosity-arg  := $(if $(VERBOSITY),--verbosity $(VERBOSITY),)
+seed-arg       := $(if $(SEED),--fixed-seed $(SEED),)
+
+dvsim-mk-args := $(waves-arg) $(verbosity-arg) $(seed-arg)
+run-dvsim     := $(dvsim-py) $(dvsim-std-args) $(dvsim-mk-args)
```

Comment:
just a heads up, but the very first argument that dvsim will expect is the `ibex_icache_sim_cfg.hjson` file, all of the other cmdline options need to be after it